webapp/client/src/pages/LoginPage.js

```diff
@@ -1,60 +1,94 @@
 import PropTypes from "prop-types";
 import React from "react";
+import { useTranslation } from "react-i18next";
+import FormFieldIdNr from "../components/FormFieldIdNr";
+import FormFieldUnlockCode from "../components/FormFieldUnlockCode";
 import FormHeader from "../components/FormHeader";
+import FormRowCentered from "../components/FormRowCentered";
 import StepForm from "../components/StepForm";
 import StepHeaderButtons from "../components/StepHeaderButtons";
 
-export default function LoginPage({
-  prevUrl,
-  backLinkText,
-  stepTitle,
-  stepIntro,
-  submitUrl,
-  overviewButton,
-  csrfToken,
-  explanatoryButtonText,
-  nextButtonLabel,
-}) {
+export default function LoginPage({ backLink, stepHeader, form, fields }) {
+  const { t } = useTranslation();
+
   return (
-    <React.Fragment>
-      <StepHeaderButtons backLinkUrl={prevUrl} backLinkText={backLinkText} />
-      <FormHeader title={stepTitle} intro={stepIntro} />
-      <StepForm
-        {...{
-          submitUrl,
-          overviewButton,
-          csrfToken,
-          explanatoryButtonText,
-          nextButtonLabel,
-        }}
-      />
-    </React.Fragment>
+    <>
+      <StepHeaderButtons {...backLink} />
+      <FormHeader {...stepHeader} />
+      <StepForm {...form}>
+        <FormRowCentered>
+          <FormFieldIdNr
+            autofocus
+            required
+            fieldName="idnr"
+            // TODO: is the fieldId ever different from the fieldName?
+            fieldId="idnr"
```

Comment:
I think it is, for example for the separated fields. The sub-fields, I think, have a different name than id?